Data/Uniques/bow.lua

```diff
@@ -40,7 +40,7 @@ Adds 25 to 50 Fire Damage
 {variant:1}5% increased Attack Speed
 {variant:2,3,4}(10-14)% increased Attack Speed
```

Comment:
Should this be added to variant 5 as well?

Reply:
Updated. Also moved dexterity so list of abilities matches what is is in game.